modin/experimental/core/execution/native/implementations/hdk_on_native/test/utils.py

```diff
@@ -272,16 +274,21 @@ def run_modin(
         kwargs["df2"] = pd.DataFrame(data2, **constructor_kwargs)
         kwargs["df"] = kwargs["df1"]
 
-        if force_lazy:
```

Comment:
By moving `force_lazy` code branch after `force_arrow_execute` code branch, you implicitly change the execution of some tests. For example, `test_empty_transform` and `test_append`.

To avoid this, you can make a check that only one of the 3 parameters (`force_lazy`, `force_hdk_execute`, `force_arrow_execute`) can be true.

Reply:
Yes and it seems correct. Previously, `force_arrow_execute` was always ignored, unless `force_lazy` was set to `False` explicitly, because it's `True` by default. The mentioned tests set `force_arrow_execute=True`, but do not set `force_lazy=False`. Thus, the `force_arrow_execute` parameter was simply ignored.